app/src/main/java/com/anwera64/pagodividido/presentation/base/BaseViewModelFragment.kt

```diff
@@ -0,0 +1,25 @@
+package com.anwera64.pagodividido.presentation.base
+
+import android.os.Bundle
+import android.view.View
+import androidx.databinding.ViewDataBinding
+import androidx.lifecycle.ViewModel
+import org.koin.androidx.scope.currentScope
+import org.koin.androidx.viewmodel.ext.android.viewModel
+import kotlin.reflect.KClass
+
+abstract class BaseViewModelFragment<V : ViewModel, B : ViewDataBinding>(type: KClass<V>) : BaseFragment<B>() {
+
+    protected val viewModel: V by currentScope.viewModel(this, type)
+    abstract val viewModelValue: Int?
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        viewModelValue?.let {
+            binding.setVariable(it, viewModel)
```

Comment:
add name to it variable

Reply:
done in base fragment and also in base activity